integration/kind-test/scripts/run-tests.sh

```diff
@@ -18,14 +17,19 @@ mkdir -p "$LOGS"
 CTL_PID=$!
 echo "controller PID:$CTL_PID"
 
-go run $SCENARIOS/runner/main.go $NAMESPACE $SERVICE $CLUSTERID1 $CLUSTERSETID1 $ENDPT_PORT $SERVICE_PORT "$endpts"
+go run $SCENARIOS/runner/main.go $NAMESPACE $SERVICE $CLUSTERID1 $CLUSTERSETID1 $ENDPT_PORT $SERVICE_PORT $SERVICE_TYPE "$endpts" 
 exit_code=$?
 
 if [ "$exit_code" -eq 0 ] ; then
   ./integration/shared/scripts/test-import.sh "$EXPECTED_ENDPOINT_COUNT" "$endpts"
   exit_code=$?
 fi
 
+if [ "$exit_code" -eq 0 ] ; then
+  ./integration/kind-test/scripts/DNS-test.sh
+  exit_code=$?
+fi
+
```

Comment:
We need to exit if either of these tests fail, otherwise it will go on to the scaling test, and final exit code will only reflect that.